pkg/k8s/json_patch.go

```diff
@@ -14,7 +14,7 @@
 
 package k8s
 
-const(
+const (
```

Comment:
I've made this into a new PR so it can be easy to be backported https://github.com/cilium/cilium/pull/7819

Reply:
dropping this once that merges

Reply:
dropped this for the current version, as CI does not care about this :-)